registry/server/routes/config.ts

```diff
@@ -72,6 +77,11 @@ router.get('/', async (req, res) => {
         return acc;
     }, {});
 
+    data.settings = preProcessResponse(settings).reduce((acc: {[key: string]: any}, setting: Setting) => ({
```

Comment:
Can we here treat dots in setting key as object levels.

So if we have `a.b` & `a.c` config endpoint should return:
```
{
  a: {b: 'XXX', c: 'XXX'}
}
```

Reply:
I decided to use `_.set()` here to have the result as you described